lab_1/main.py

```diff
@@ -2,16 +2,28 @@
 Lab 1
 Language detection
 """
+import string
 
 
 def tokenize(text: str) -> list or None:
     """
     Splits a text into tokens, converts the tokens into lowercase,
     removes punctuation and other symbols from words
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
+        :param text: a text
+        :return: a list of lower-cased tokens without punctuation
     """
-    pass
+    text_new = ""
+    if isinstance(text, str):
```

Comment:
put checks with None to the beginning
it's easier to read

if the arguments are incorrect, we return None right at the beginning